lib/house.rb

```diff
@@ -0,0 +1,35 @@
+class House
+
+  attr_reader :price, :address, :rooms
+  def initialize(price, address)
+    @price = price
+    @address = address
+    @rooms = []
+  end
+
+  def add_room(room)
+    @rooms << room
+  end
+
+  def rooms_from_category(room_category)
+    @rooms.find_all {|room| room.category == room_category}
+  end
+
+  def area
+    area = 0
+    @rooms.each {|room| area = area + room.area}
+    area
```

Comment:
Great! This works well.  Ruby has shorthand for adding and reassigning a value, so you could have written your line 20 like this: 

`@rooms.each {|room| area += room.area}`


Ruby also has an enumerable called `sum` which could have made this whole method just one line. 

`@rooms.sum {|room| room.area}`